conf/routes

```diff
@@ -20,7 +20,6 @@ POST        /digital/country                 controllers.DigitalPack.handleCount
 # Checkout
 GET         /checkout                        controllers.Checkout.renderCheckout
 POST        /checkout                        controllers.Checkout.handleCheckout
-GET         /checkout/thankyou               controllers.Checkout.thankyou
```

Comment:
I am afraid query parameters wouldn't cut it, as we now have a number of (nested) fields to display:

- subscription id
- personal data (i.e. name with surname, postal address, email address)
- when we create a guest user, also a password form (i.e. Identity user id and token)

If we really have a requirement for this page to have a permanent URL, than I am afraid we need to store the data above in a database record, or add significant code to fetch all the relevant bits of information from their respective source based on a single user identifier  (i.e email). Either ways, it seems to me that at this stage that would involve a fair amount of work and add little business value.